CHANGELOG.md

```diff
@@ -47,11 +48,34 @@ is added to that terminator. This will bias future dials towards other terminato
 
 The failure cost can be reduced by successful dials. Failure cost is also reduced over time. In previous
 releases this was a fixed credit of 5, every minute. This is now changing to an exponential 
-amount, based on time since the last failure. The credit is now: `2 ^ (minutes since last failure/5)`.
+amount, based on time since the last failure. 
+
+If X is minutes sincew last failure, the credit is now: `min(10, 2 ^ (X/5))`.
+
+## Router Metrics Change
+
+Router metrics has a new configuration setting:
+
+```
+metrics:
+  # Number of usage events to be able to queue. Defaults to 256. If this queue backs up, it can
```

Comment:
defaults to 256 or 1024? are these mismatched?

Reply:
Updated. I sometimes think that it doesn't make sense to show the setting as the default value, since you don't need it in that case, but I concede that possibility of confusion by having different numbers outweighs that minor consideration.